src/inference/include/openvino/runtime/intel_gpu/ocl/dx.hpp

```diff
@@ -46,8 +46,9 @@ class D3DBufferTensor : public ClBufferTensor {
      */
     static void type_check(const Tensor& tensor) {
         RemoteTensor::type_check(tensor,
-                                 {{ov::intel_gpu::dev_object_handle.name(), {}},
-                                  {ov::intel_gpu::shared_mem_type.name(), {ov::intel_gpu::SharedMemType::DX_BUFFER}}});
+                                 {{std::string(ov::intel_gpu::dev_object_handle.name()), {}},
+                                  {std::string(ov::intel_gpu::shared_mem_type.name()),
+                                   {ov::Any(ov::intel_gpu::SharedMemType::DX_BUFFER).as<std::string>()}}});
```

Comment:
why do we need to wrap with `ov::Any`?

Reply:
Here we have a map string to vector of strings. So it is why I use any to covert from enum to string.